src/main/java/uk/gov/gchq/palisade/resource/request/GetResourcesByTypeRequest.java

```diff
@@ -39,21 +40,23 @@ public GetResourcesByTypeRequest() {
      * @return the {@link GetResourcesByTypeRequest}
      */
     public GetResourcesByTypeRequest type(final String type) {
```

Comment:
This needs the @generated annotation on the mutator

Reply:
Done

Reply:
done